commons-geometry-core/src/test/java/org/apache/commons/geometry/core/partitioning/bsp/AbstractPartitionedRegionBuilderTest.java

```diff
@@ -38,9 +38,7 @@ public void testCtor_invalidTree() {
         final TestRegionBSPTree tree = new TestRegionBSPTree(true);
 
         // act/assert
-        GeometryTestUtils.assertThrowsWithMessage(() -> {
-            new TestRegionBuilder(tree);
-        }, IllegalArgumentException.class, "Tree must be empty");
+        GeometryTestUtils.assertThrowsWithMessage(() -> new TestRegionBuilder(tree), IllegalArgumentException.class, "Tree must be empty");
```

Comment:
I disagree with this change. The braces on the lambda are of course not needed, but they were added for readability. With the original multi-line form, the statement being tested is immediately visible. In the form proposed here, the reader must scan the method arguments in order to determine what is actually being tested. This is not so bad in this particular case, but in others it becomes quite difficult to see what is happening. Please revert this change and others like it.

Reply:
reverted

Reply:
This comment applies to all updates of `GeometryTestUtils.assertThrowsWithMessage()`, not just this instance.

Reply:
done